### PR TITLE
Update the code review docs

### PR DESCRIPTION
**Pull request by johnpaulashenfelter**
Clean, reduce redundancy, and update.

Addresses #507

### CODE REVIEW

File: platform/engineering/code_review_guidelines.md [hunk 1]
```diff
@@ -4,41 +4,58 @@ This document covers rules for engaging the VSP team in [pull request reviews](h
...
-
-* **Size/complexity**: keep your changes scoped to one feature at a time. This means:
-    * **Size**: changes should be smaller than **300** changed lines (not including tests or whitespace). *If your change can't be smaller than this, talk to the team in #vfs-platform-support before asking for a review.*
+We expect that most developers have worked with GitHub and review processes before, but want to be clear about our norms to reduce confusion and help code flow through the process as quickly and reliably as possible.
+
+
+## The pull request review lifecycle in brief
+
+1. For initial review by your project team, create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
+2. Once you are ready for the VSP team to review your PR, use the GitHub *Ready for review* button in the merge section of your pull request. An engineer should review the code within 2 days.
```
Comment (gunsch) [inline platform/engineering/code_review_guidelines.md]: this seems like a policy change, not just a guidelines/wording update (1 day --> 2 days), was that intentional?

File: platform/engineering/code_review_guidelines.md [hunk 2]
```diff
@@ -4,41 +4,58 @@ This document covers rules for engaging the VSP team in [pull request reviews](h
...
+
+When your code is reviewed, here are some of the things we're looking for:
+
+* *Correctness*. Does the code correctly implement the described feature?
+* *Code quality*. Is the code readable? Is the code language-idiomatic?
+* For front end changes, are screenshots of the change(s) included?
+
+
+
+## Helpful hints
```
Comment (gunsch) [inline platform/engineering/code_review_guidelines.md]: Can we pick a different title? This doc should be *requirements* we're imposing on teams sending things to us, not *advice*, and headings/wording should reflect that.

File: platform/engineering/code_review_guidelines.md [hunk 3]
```diff
@@ -4,41 +4,58 @@ This document covers rules for engaging the VSP team in [pull request reviews](h
...
-
-*Hint*: smaller, more frequent pull requests will make this easier for everyone!
-
-## Guidelines
-
-* **Review lifecycle**
-    * Before sending code to VSP team have it reviewed by your team (see **First review** below) using a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).  
-      * This prevents [code owners](https://help.github.com/en/articles/about-code-owners) from being notified before pull request is ready for review.
-    * After having your pull request reviewed by your team you can click the *Ready for review* button in the merge section of your pull request.
-      * An engineer from the VSP team will aim to review within 24 hours (business days) after the pull request has been reviewed by your team and you have been marked it *Ready for review*. Each additional round of changes will be reviewed within 24 hours.  If you need a review faster, escalate via your DSVA product owner.
```
Comment (gunsch) [inline platform/engineering/code_review_guidelines.md]: this update also loses the clarity about "additional rounds of changes", which IMO we should keep explicitly called out

File: platform/engineering/code_review_guidelines.md (hunk 1 above)
Comment (johnpaulashenfelter) [reply platform/engineering/code_review_guidelines.md]: 1 business day like we discussed on slack

File: platform/engineering/code_review_guidelines.md (hunk 2 above)
Comment (johnpaulashenfelter) [reply platform/engineering/code_review_guidelines.md]: Done.

File: platform/engineering/code_review_guidelines.md (hunk 3 above)
Comment (johnpaulashenfelter) [reply platform/engineering/code_review_guidelines.md]: Thanks! That literally just got dropped in edit.

**Review by gunsch — APPROVED** [review]
LGTM, added @annaswims as a reviewer to take a look as well.

File: platform/engineering/code_review_guidelines.md [hunk 4]
```diff
@@ -4,41 +4,55 @@ This document covers rules for engaging the VSP team in [pull request reviews](h
...
-    * **Does it have screenshots of the change?** (FE changes)
-
-* **Description**: make it clear what the change is describing.
-    * **PR title** should be a concise summary of the change.
-    * In your **PR description**, discuss how code changes are related to the pull request objectives.
-    * Reference the **GitHub issues** your PR relates to, in order to provide reviewers more context.
-
-* **Size/complexity**: keep your changes scoped to one feature at a time. This means:
-    * **Size**: changes should be smaller than **300** changed lines (not including tests or whitespace). *If your change can't be smaller than this, talk to the team in #vfs-platform-support before asking for a review.*
+We expect that most developers have worked with GitHub and review processes before, but want to be clear about our norms to reduce confusion and help code flow through the process as quickly and reliably as possible.
```
Comment (annaswims) [inline platform/engineering/code_review_guidelines.md]: FWIW, Github is still relatively new to VA culture and git is not necessarily the dominant form of source control at VA https://dsva.slack.com/archives/C04KXPXL4/p1568909161000900

Comment (gunsch) [reply platform/engineering/code_review_guidelines.md]: +1, but also it's still OK to state that as the _expectation_ for working with VSP

(maybe we can word away from "most  developers have worked with GitHub", but "we expect developers to be familiar with GitHub" is still true)

Comment (johnpaulashenfelter) [reply platform/engineering/code_review_guidelines.md]: Thanks -- solid point to education me on 😄

**Pull request merged**